### DISCUSSION TITLE
Completion panel not opening in Codespace when i click in three dots

### DISCUSSION THREAD

**Discussion by bhavishyasingla1** [skills/.github#613] [Code with Copilot]
![image](https://github.com/skills/.github/assets/113583143/c8a02746-c976-47b5-87b6-ac19974b2421)
![image](https://github.com/skills/.github/assets/113583143/0e61fe19-4315-4dd6-ade9-c1ed0a2029ca)

**Comment by rpanekwtw** [comment]
I had to hit space then hover over the grey { to get the popup.  Hovering over the red squiggly as suggested brought something else up

![image](https://github.com/user-attachments/assets/07829000-51f8-4691-8b37-36e09ea5f485)

**Reply by Gohfe** [reply]
# - [ ]
